src/libratbag-enums.h

```diff
@@ -342,6 +342,7 @@ enum ratbag_led_type {
 	RATBAG_LED_TYPE_SIDE,
 	RATBAG_LED_TYPE_BATTERY,
 	RATBAG_LED_TYPE_DPI,
+	RATBAG_LED_TYPE_SCROLL_WHEEL,
```

Comment:
looking at this, I think we should just call it 'wheel'. The physical types are an aid but don't need to be 100% accurate (see also side vs left/right side) and 'wheel' is more generic than scroll wheel - and less to type. With that, ack by me for this one.


also, please make sure this commit is before the other one that uses the new type